### PR TITLE
Test

### PR DESCRIPTION
**Pull request by joel-kalema**
- [ ] This is a pair-programming project. Please decide whose project will you use for testing.
- [ ] Use jest framework for testing.
- [ ] Create a test file (`[..].test.js`) for a file containing the add item and delete item functions that you want to test.
- [ ] Make sure you group your tests using the `describe()` method.

### CODE REVIEW

**Review by FrederickMih — COMMENTED** [review]
Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project with Linters not properly configured. Please configure linters in your project and submit it once more.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by wuletawwonte** [comment]
## Hi @FrederickMih  
We have fixed the linter errors. Please review our project now.

File: src/modules/TodoTask.test.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    kk.addTask('add the task');
+    expect(kk.tasks).toHaveLength(lengthArry + 1);
+  });
+  test('remove a task', () => {
+    kk.addTask('add the task');
+    const lengthArry = kk.tasks.length;
+    kk.removeList(1);
+    expect(kk.tasks).toHaveLength(lengthArry - 1);
+  });
+});
```
Comment (keroloussamy) [inline src/modules/TodoTask.test.js:19]: - [ ] Kindly you have to add tests for the HTML and for the local storage as the requirements asked.
You will have to use mocks to test if add/delete functions add or remove exactly one `<li>` element to/from the list in the DOM and mock to test local storage.

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/TodoTask.test.js (hunk 1 above)
Comment (wuletawwonte) [reply src/modules/TodoTask.test.js:19]: ## Hi @keroloussamy 
Yeah, thanks for the review. I wanted to point out something, Our addTask and removeList functions are "Pure functions". Which means there are no side effects. Which means the functions don't manipulate the DOM or don't update the localStorage. So, I think We aren't supposed to use Mocks and test HTML and local storage

**Review by brytebee — APPROVED** [review]
Hi, Team👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/TodoTask.test.js (hunk 1 above)
Comment (keroloussamy) [reply src/modules/TodoTask.test.js:19]: Hello, @wuletawwonte it seems it's a mistake from me, I noticed that your functions are pure, but I wanted to let you try Mocks and test your Html and local storage, but your project is approved already so you are right. 
And sorry again 😄

**Pull request merged**